_stylesheets/docs.css

```diff
@@ -756,8 +756,8 @@ table > tbody > tr > td > div > div > p > code {
 }
  table tr th, table tr td {
      padding: 0.5625em 0.625em;
-     font-size: inherit;
-     color: #333333;
+     font-size: 14px;
```

Comment:
Could alternatively make this `0.92em` to match the size for table cell entries (slightly smaller).

Reply:
Looks like this:

![image](https://user-images.githubusercontent.com/3442316/87089920-617d3d00-c1f4-11ea-96f3-7c8b3d45f375.png)